src/pages/photo-analysis.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { Layout } from "../components/Layout"
```

Comment:
This might be a silly question, but why do we need this page? (not sure if this is just how Gatsby works, but I thought we just have one index page?)

Reply:
This is true.... to be fixed later I guess :)